Mathlib/Probability/Moments.lean

```diff
@@ -285,6 +286,28 @@ theorem iIndepFun.cgf_sum {X : ι → Ω → ℝ}
   · rw [h_indep.mgf_sum h_meas]
   · exact (mgf_pos (h_int j hj)).ne'
 
+theorem mgf_ident_distrib
+    (X : Ω → ℝ) {Ω' : Type u_1} {m' : MeasurableSpace Ω'} {μ' : Measure Ω'} (X' : Ω' → ℝ)
+    (hident : IdentDistrib X X' μ μ') (t : ℝ) :
+    mgf X μ t = mgf X' μ' t := by
+  rw [mgf, mgf]
+  apply IdentDistrib.integral_eq
+  let u := fun x => Real.exp (t * x)
+  have compX : (fun ω ↦ Real.exp (t * X ω)) = u ∘ X := rfl
+  have compX' : (fun ω ↦ Real.exp (t * X' ω)) = u ∘ X' := rfl
+  rw [compX, compX']
+  exact IdentDistrib.comp hident (Measurable.exp (measurable_const_mul t))
+
+theorem mgf_sum_iid
+    {X : ι → Ω → ℝ}
+    (h_meas : ∀ (i : ι), Measurable (X i))
+    (h_indep : ProbabilityTheory.iIndepFun (fun _ => inferInstance) X μ)
+    (hident : ∀ (i j : ι), ProbabilityTheory.IdentDistrib (X i) (X j) μ μ)
+    (s : Finset ι) (j : ι) : ∀ t : ℝ, mgf (∑ i ∈ s, X i) μ t = (mgf (X j) μ (t)) ^ s.card := by
```

Comment:
```suggestion
    (s : Finset ι) (j : ι) (t : ℝ) : 
    mgf (∑ i ∈ s, X i) μ t = (mgf (X j) μ t) ^ s.card := by
```
Put `t` left of the colon, in this way you don't need to `intro` it in the proof below.